.github/workflows/create_test_patches.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       #        prior version (18.17.1) to see if it stabilizes
       - uses: actions/setup-node@v4
         with:
-          node-version: 18.17.1
+          node-version: 18.18
```

Comment:
I think I had it pinned to a specific version because there was some problem before with 18.18 and 18.19 but with 18.20+ I think everything worked, better to change this to just be "18" I think, or, even better run one with "20" to see if it works since 20 is current stable